src/cloudWatchLogs/commands/searchLogGroup.ts

```
@@ -66,31 +67,26 @@ export async function prepareDocument(
     uri: vscode.Uri,
     logData: CloudWatchLogsData,
     registry: LogDataRegistry
-): Promise<Result> {
+) {
```

throw instead of return